### PR TITLE
1031 update team member area

### PR DESCRIPTION
**Pull request by jonathanbobel**
Altered the team members area to have more of a card layout instead of listed down the page so it's easier to scan

### CODE REVIEW

File: tests/app/main/views/test_templates.py [hunk 1]
```diff
@@ -631,8 +631,8 @@ def test_should_show_sms_template_with_downgraded_unicode_characters(
     mock_get_template_folders,
     fake_uuid,
 ):
-    msg = "here:\tare some “fancy quotes” and zero\u200Bwidth\u200Bspaces"
-    rendered_msg = "here: are some “fancy quotes” and zerowidthspaces"
+    msg = "hey"
+    rendered_msg = "hey"
```
Comment (ccostino) [inline tests/app/main/views/test_templates.py:635]: What happened with this change?  Was the test failing for some reason?

**Review by ccostino — COMMENTED** [review]
Thanks, @jonathanbobel!

I had one question on one of the test changes that caught my eye; it was also a little weird to see the names change to not have a space between the name and the parentheses, but not a showstopper since it's test data. :-)

**Comment by ccostino** [comment]
@jonathanbobel looks like this needs an update with `main` in addition to the questions I had - resending expired invites are now in place! :-)

File: tests/app/main/views/test_templates.py (hunk 1 above)
Comment (jonathanbobel) [reply tests/app/main/views/test_templates.py:635]: Let me look back and check, I don't think I meant to have that as the true "fix" - if I remember correctly I was having a few issues with the unicode character being the same between msg and rendered_msg

**Pull request merged**